server/models/Notification.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const notificationSchema = new Schema({
+    
+    receiver: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
```

Comment:
Won't this be a required attribute?

Reply:
I do not know why I always forget the required attribute 😂.